src/SignUpIn/DelivererGroup/DelivererGroupSignUpController.js

```diff
@@ -0,0 +1,99 @@
+import '../SignUpIn.css';
+import DelivererGroupSignUp from './DelivererGroupSignUp';
+import DelivererGroupSignUp1 from './DelivererGroupSignUp1';
+import DelivererGroupSignUp2 from './DelivererGroupSignUp2';
+import SignUpComplete from '../SignUpComplete';
+
+let React = require('react');
+let createReactClass = require('create-react-class');
+
+let fieldValues = {
+    organizationName: null,
+    numVolunteers: null,
+    address1: null,
+    address2: null,
+    city: null,
+    state: null,
+    zip: null,
+    
+    email: null,
+    password: null,
+    contactName: null,
+    contactPosition: null,
+    contactEmail: null,
+    contactNumber: null,
+}
+
+let DelivererGroupSignUpController = createReactClass({
+    getInitialState: function () {
+        return {
+            step: 0
+        }
+    },
+    saveValues: function (fields) {
+        return function () {
+            // Remember, `fieldValues` is set at the top of this file, we are simply appending
+            // to and overriding keys in `fieldValues` with the `fields` with Object.assign
+            // See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign
+            fieldValues = Object.assign({}, fieldValues, fields)
+        }()
+    },
+
+    nextStep: function () {
+        this.setState({
+            step: this.state.step + 1
```

Comment:
Depending on the values of `this.state` inside `setState()` will not work as intended all the time. This goes for everywhere you're doing this. Read: https://reactjs.org/docs/faq-state.html#why-is-setstate-giving-me-the-wrong-value